setup-scripts/ubuntu-setup.sh

```diff
@@ -0,0 +1,115 @@
+#!/bin/bash
+
+# Run as root
```

Comment:
Does this work for ubuntu-20, 22.04 and 24.04 ?

Reply:
Currently it seems like this script only works for 22.04. I tested this on 24.04 ami and seems like there were some apt packages mismatch. I have updated the name of this setup script to imply that this is only for 22.04. We can create similar scripts for different build systems or upgrade this script to have some conditional logic based on build systems. This can be its own line item though in my opinion.